docs/dev/README.md

```diff
@@ -69,9 +55,32 @@ processing.
   access them.
   - **Data Folders** - A type for managing data and metadata in a local data folder, an Amazon S3 bucket, or an
   Microsoft Azure Blob Storage container.
+  - **Error** - Error type used throughout the crate, a single error type is used for simplicity.
   - **Manager** - Manages metadata related to the ModelarDB manager and provides functionality for interacting with the
   ModelarDB manager.
   - **Remote** - A public interface to interact with the ModelarDB DBMS server using Apache Arrow Flight.
+- [modelardb_storage](/crates/modelardb_storage) - Library providing functionality for reading from and writing to
+storage.
+  - **Metadata** - Manages metadata stored in Delta Lake, e.g., information about the tables' schema and compressed
+  data.
+  - **Optimizer** - Rules for rewriting Apache DataFusion's physical plans for model tables so aggregates are computed
+  from compressed segments instead of from reconstructed data points.
+  - **Query** - Types that implement traits provided by Apache DataFusion so SQL queries can be executed for ModelarDB
+  tables.
+  - **Delta Lake** - Module providing functionality for reading from and writing to a delta lake.
+  - **Error** - Error type used throughout the crate, a single error type is used for simplicity.
+  - **Parser** - Extensions to Apache DataFusion's SQL parser. The first allow it to create model tables with a
```

Comment:
allow*s* ? (if it is one extension that does that)

Reply:
Fixed.